### ISSUE TITLE
Concatenation of the base class softmax predictions

### ISSUE THREAD

**Issue by gordon-lim** [hiarindam/document-image-classification-TL-SG#9]
Dear author,

Please allow me to clarify some details regarding the implementation. To run inference on an image using your proposed "MLNN based stacking of holistic & region-based models with inter and intra-domain weights transfer" method, I would first run each of the 5 base models (with your given weights) on the image and get 5 different "base class softmax predictions". I would then concatenate those 5 base class softmax predictions... but since each softmax prediction has 16 scores wouldn't their concatenation have a total of 5*16 scores. Afterwards, I am supposed to feed those scores to an MLNN to get the final prediction. Is that right? Do you also have the weights for the MLNN? Or do I have to train it on the validation set myself?

Please help me as I would like to be able to reproduce your results (92.21% accuracy) for my research project.

Yours sincerely,
Gordon
